linkerd/identity/Cargo.toml

```diff
@@ -8,13 +8,17 @@ edition = "2018"
 [features]
 default = []
 test-util = []
+rustls-tls = ["ring", "rustls"]
+openssl-tls = ["openssl"]
+boring-tls = ["boring"]
 
 [dependencies]
 linkerd-dns-name = { path = "../dns/name" }
-ring = "0.16.19"
-rustls = "0.19"
+ring = { version = "0.16.19", optional = true, features = ["std"] }
```

Comment:
I added the `features = ["std"]` in order to satisfy `thiserror::Error` not sure if we want this around but I'll circle back to it.

Reply:
this is fine; linkerd is always built with the Rust standard library, so enabling `ring`'s dependency on `std` is fine.